src/Domain/Aggregate/PathAggregate/PathAggregateInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace PhpTuf\ComposerStager\Domain\Aggregate\PathAggregate;
+
+/**
+ * Aggregates path value objects.
```

Comment:
"Aggregate" is a technical term that has specific meaning in Domain Driven Design (which I'm rather strictly observing). It would be redundant to explain an established pattern every time I use it. But as always, if it makes you wonder, it'll make others wonder, too... I'll think about whether there's a good way to _centrally_ define terms and explain the overall organizing design (with links to published materials). Thanks.